langs/es.ini

```diff
@@ -1,11 +1,19 @@
-name="Español"
+name="Alemán"
 dir="ltr"
-missing="Lamentableme no hay traduccion disponible"
+missing="Lamentablemente no hay ninguna traducción disponible"
 readmore="Leer más"
+languages="Idiomas"
+section="Sección"
+shortcuts="Abreviaturas"
+legal="legal"
```

Comment:
Is this an actual translation or has transifex just copied the original string?

Reply:
This is an actual translation.